### ISSUE TITLE
KPI Request: Number of Defects Found

### ISSUE THREAD

**Issue by MichaelClifford** [aicoe-aiops/ocp-ci-analysis#154]
Identify the number of defects found in the testgrid for each test. We will define "defects" as reported bugs. So we should be able to calculate the number of tests each day that have that have a non-empty value for the the "linked bug" field in the additional testgrid data.

We also need to confirm if it makes sense to capture this value uniquely for for each test on each grid, or if the linked_bugs is constant per named test.  

This card depends on #144 

**Acceptance Criteria:**

- [ ] notebook that returns the daily value for known defects 
- [ ] determine if this value is unique per test and grid, or just per named test.

**Comment by MichaelClifford** [comment]
This data might not be present in the testgrid data despite having a place for it in their schema.

![image](https://user-images.githubusercontent.com/4494906/111337529-8c097a80-864c-11eb-9fcb-81b43665bc6b.png)
 
I went ahead and did a small test to explore the available data before working on the metric notebook, unfortunately it looks like this data is not actually recorded by testgrid.   

Is this something others working on #144 have noticed?

**Comment by oindrillac** [comment]
@MichaelClifford yes, even as a part of the exploration in #144, this was found to be empty

**Comment by MichaelClifford** [comment]
Thanks for confirming @oindrillac . 

Looks like we will have to find a different method to link the tests and bugs then : (
